integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/HyperfoilResource.java

```diff
@@ -66,6 +68,17 @@ public static boolean containsFailedRunPhase(String idRun) {
         return run.getPhases().stream().anyMatch(Phase::getFailed);
     }
 
+    public static void storeBenchmarkReport(String perfTestName, String runId) {
+        try {
+            String generateReport = validation.generateReport(validation.getRun(runId));
+            Path destination = Paths.get(BENCHMARK_REPORT_DESTINATION_FOLDER_PATH, perfTestName + "-" + LocalDateTime.now() + ".html");
```

Comment:
maybe it's useful to add the `runId` param as part of the name of the generated html file, wdyt?